source/filesystem.c

```diff
@@ -196,6 +196,17 @@ void addExecutableToMenu(menu_s* m, char* execPath)
 	addMenuEntryCopy(m, &tmpEntry);
 }
 
+void loadDescriptorNetloaderApp(descriptor_s* d, char* execPath) {
```

Comment:
None of these other functions use `static` so I see why you didn't use it, but it might help the optimizer to make this function `static`.